src/pages/JoinWebinarPage/JoinWebinarPage.tsx

```diff
@@ -178,7 +175,7 @@ const JoinWebinarPage = () => {
         )}
         {!loading && currentSection === Section.WatchParty && (
           <MainStageWithChatBox>
-            <WebinarMainStage />
+            <MainStageWrapper isWebinarStage />
```

Comment:
Would this ever need to be wrapped in a `Suspense`? (I was unaware of React's capability to lazy load...)